keps/sig-node/4639-oci-volume-source/kep.yaml

```diff
@@ -46,7 +46,7 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.33"
+latest-milestone: "v1.34"
```

Comment:
Just as a note on timing:

* 1.34 enhancements freeze is June 19 (https://github.com/kubernetes/sig-release/pull/2780)
* 1.34 release is scheduled for August 27 (https://github.com/kubernetes/sig-release/pull/2780)
* containerd 2.2 (the next minor) is currently scheduled for November 5 (https://containerd.io/releases/#current-state-of-containerd-releases)

This means containerd most likely will not have support for rw OCI volumes at 1.34 enhancements freeze or by the release.  Since the current proposal reads as if the behavior is up to the runtime (rather than specified by the user in the pod spec), this may not be a problem, but I did want to call it out.

Reply:
Thanks! Yes you're right, this proposal change is up to the runtime implementation. :+1: